.github/workflows/build.yaml

```diff
@@ -15,7 +15,8 @@ jobs:
     name: Build
     strategy:
       matrix:
-        python-version: [ '3.9', '3.10', 'pypy3.9' ]
+        #python-version: [ '3.9', '3.10', 'pypy3.9' ]
```

Comment:
is there a reason we are changing which python versions are supported? Was this a local change that you didn't mean to push?